src/models/results.ts

```diff
@@ -115,6 +115,8 @@ export class AstResult extends CxResult {
 
   getIcon() {
     switch (this.severity) {
+      case "CRITICAL":
```

Comment:
replace with constant (it will be great if you can do it to all the severities here)

Reply:
Will do